styles/nitrotype/catppuccin.user.css

```diff
@@ -0,0 +1,1421 @@
+    }
+    /* green */
+    .btn.is-on,
+    .is-on.modal--bundle-offer--purchase-option,
+    .is-on.friends-list--friend-status--join-race-button,
+    .btn.is-on:focus,
+    .is-on.modal--bundle-offer--purchase-option:focus,
+    .is-on.friends-list--friend-status--join-race-button:focus {
+      color: @green;
+    }
+    /* btn secondary "Return to Garage" */
+    /* switch box */
+    .switch {
+      background: rgba(@crust, 0.5);
+      border-color: @border;
+    }
+    /* badges */
+    .rarity-badge {
+      color: @colored-button-text;
+    }
+    /* popups */
+    /* growl (notify) */
+    /* body */
+    .growl--a {
+      background-color: @overlay2;
+      color: @text;
+    }
+    /* team name */
+    span.growl-user > span {
+      color: @blue !important;
+    }
+    /* close btn */
+    .growl-close {
+      background-color: @red;
+      border-color: @red;
+      color: @colored-button-text;
+    }
+    /* logout popup */
+    .modal--error .modal-container {
+      background: @red !important;
+      border-color: @red;
+    }
+    /* "Yes, I'm here! btn */
+    .btn--light {
+      background-color: @mantle;
+      color: @text;
+    }
+    /* text generic */
+    .tc-i {
+      color: @text;
+    }
+    /* cash popup claim */
+    .modal-close {
+      background-color: @accent-color;
+      color: @colored-button-text;
+    }
+    .modal--a .modal-container {
+      background-color: @crust;
+    }
+    /* loading */
+    /* loader spin wheel */
+    /* wheel balls */
+    .loader--default .loader-animBall {
+      color: @blue;
+    }
+    /* loading text */
+    .loader--default .loader-text {
+      color: @text;
+    }
+    /* notifications */
+    /* friend online notifications */
+    .notification--friend-online--info {
+      background: rgba(@overlay0, 0.9);
+      color: @text;
+      text-shadow: 0 1px 0 @crust;
+      box-shadow:
+        inset 0 0 0 1px rgba(255, 255, 255, 0.15),
+        0 2px 11px rgba(@crust, 0.3);
+    }
+
+    #root {
+      /* loading overlay */
+      > div:not(.structure, .growls) {
+        background-color: @base !important;
+      }
+      /* center modal */
+      > div:not(.structure) {
+        > div:not(.modal-container) {
+          background-color: @mantle !important;
+        }
+        /* "loading..." text */
+        > div > div:nth-child(2) {
+          color: @text;
+        }
+      }
+    }
+
+    /* NT logo, NT logo on the home page, NT logo on the loading screen */
+    div.header-core > div > a > svg,
+    div.header-core.g.g--flag.g--f > div.g-b.g-b--6of12 > div > a > svg,
+    #root > div > div > div:nth-child(1) > svg {
+      /* the first letter "N" */
+      > path:nth-child(7) {
+        fill: @accent-color !important;
+      }
+      /* the rest of the letters */
+      > path {
+        fill: @text !important;
+      }
+      /* the  slash at the end of the logo */
+      > path:nth-child(1) {
+        fill: @accent-color !important;
+      }
+    }
```

Comment:
Let's make this cleaner please, we can target things like `.logo-SVG` and `[fill="#D62F3A"]` and `.logo-deco` and `.logo-letter`, etc . I can't seem to capture the loading state HTML though, what does it look like?